app/pipedrive/_schema.py

```diff
@@ -136,26 +138,29 @@ async def company_dict(self) -> dict:
 class Person(PipedriveBaseModel):
     id: Optional[int] = Field(None, exclude=True)
     name: str
-    primary_email: Optional[str] = ''
+    primary_email: Optional[str] = Field('', custom=True)
     phone: Optional[str] = ''
-    address_country: Optional[str] = None
+    address_country: Optional[str] = Field(None, custom=True)
```

Comment:
I don't think either of these are custom fields? Especially email?

Reply:
you are correct on email, we need `address_country` tho